dolphin/utils.py

```diff
@@ -709,3 +709,13 @@ def __call__(cls, *args, **kwargs):
                 if cls not in cls._instances:
                     cls._instances[cls] = super(Singleton, cls).__call__(*args, **kwargs)
         return cls._instances[cls]
+
+
+def bool_from_string(str, default=False):
+    str = str.lower()
+    if str == 'true':
```

Comment:
How about str.Capitalize()

Reply:
IMO, both fine, what we saved in db is bool not string.

Reply:
Ok. I was asking for using python func